app/assets/javascripts/IndividualResult.js

```diff
@@ -0,0 +1,46 @@
+const mongoose = require('mongoose');
+
+const [mNumber, mString, Mixed, ObjectId] = [
```

Comment:
Any particular reason for appending these with `m`?

Reply:
@ljades brought this up as well, so I'll tag him in the explanation: at one point in the development process for some of the other models, I was running into clashes with the Javascript builtins for `Number` `String`, etc., so I appended the `m` to avoid bumping into those namespaces. It's possible those issues were coming from an unrelated source, but when I was working on that, using the `m` fixed them, so I decided to duplicate it here. I can swap them all out for their un-appended versions and see what it breaks.

Reply:
Agreed, I think that is the way to go.

Reply:
Ok, I'll revert these, and we can bring them back in another PR.